datanode/gateway/graphql/schema.graphql

```diff
@@ -3141,6 +3143,9 @@ type Vote {
 
   "The weight of this vote based on the total of governance token"
   governanceTokenWeight: String!
+
+  "The weight of this vote based om the total equity like share"
```

Comment:
```suggestion
  "The weight of this vote based on the total equity like share"
```

Reply:
Is this based on the total equity-like share that the public key that's voting has?